src/main/pages/che-7/administration-guide/assembly_deploy-kubernetes-image-puller-on-k8s-with-helm.adoc

```diff
@@ -0,0 +1,89 @@
+|`configMap.cachingMemoryRequest`	|The value of `CACHING_MEMORY_REQUEST` to be set in the ConfigMap	|`"10Mi"`
+|`configMap.cachingMemeryLimit`	|The value of `CACHING_MEMORY_LIMIT` to be set in the ConfigMap	|`"20Mi"``
+|`configMap.nodeSelector`	|The value of `NODE_SELECTOR` to be set in the ConfigMap |`"{}"`
+|======
+
+=== Installation
+First, clone the `kubernetes-image-puller` repository: 
+
+`git clone https://github.com/che-incubator/kubernetes-image-puller && cd kubernetes-image-puller`
+
+Then, create a namespace to deploy the puller into:
+
+`kubectl create namespace k8s-image-puller`
+
+Run `helm install`:
+
+`helm install kubernetes-image-puller -n k8s-image-puller deploy/helm`
+
+To set values, change `deploy/helm/values.yaml` or use `--set property.name=value`
+
+=== Verify Installation
+After installation, you should see a deployment, and a `kubernetes-image-puller` deployment, and daemonset named `DAEMONSET_NAME` with a pod for each node in your cluster:
+
+```
+$ kubectl get deployment,daemonset,pod -n k8s-image-puller
+NAME                                      READY   UP-TO-DATE   AVAILABLE   AGE
+deployment.apps/kubernetes-image-puller   1/1     1            1           4m21s
+
+NAME                                     DESIRED   CURRENT   READY   UP-TO-DATE   AVAILABLE   NODE SELECTOR   AGE
+daemonset.apps/kubernetes-image-puller   3         3         3       3            3           <none>          3m40s
+
+NAME                                          READY   STATUS    RESTARTS   AGE
+pod/kubernetes-image-puller-b556c8bd4-qxwwl   1/1     Running   0          4m21s
+pod/kubernetes-image-puller-glh87             3/3     Running   0          3m40s
+pod/kubernetes-image-puller-h7lxf             3/3     Running   0          3m40s
+pod/kubernetes-image-puller-nrt6p             3/3     Running   0          3m40s
+```
+
+Check that the `ConfigMap` named `k8s-image-puller` has the values you specified in your `helm` configuration, or that they contain the default values:
+
+```
+$ kubectl get configmap k8s-image-puller -o yaml
+apiVersion: v1
+data:
+  CACHING_INTERVAL_HOURS: "1"
+  CACHING_MEMORY_LIMIT: 20Mi
+  CACHING_MEMORY_REQUEST: 10Mi
+  DAEMONSET_NAME: kubernetes-image-puller
+  IMAGES: java11-maven=quay.io/eclipse/che-java11-maven:nightly; che-theia=eclipse/che-theia:next;
+    java-plugin-runner=eclipse/che-remote-plugin-runner-java8:latest;
+  NAMESPACE: k8s-image-puller
+  NODE_SELECTOR: '{}'
+kind: ConfigMap
+metadata:
+  creationTimestamp: "2020-02-17T22:15:22Z"
+  name: k8s-image-puller
+  namespace: k8s-image-puller
+  resourceVersion: "3078"
+  selfLink: /api/v1/namespaces/k8s-image-puller/configmaps/k8s-image-puller
+  uid: 71bc5ce4-d095-468e-ab7b-23c1e0a36638
+```
+
+When creating a workspace that relies on cached images specified in `IMAGES`, you should see Eclipse Che confirm that the images are already present when the workspace is starting:
+
+image::installation/image-already-present-on-machine.png[]
```

Comment:
As with the OpenShift doc, I'm not sure this message will always be present in k8s.